dashboard/app/views/levels/editors/_studio.html.haml

```diff
@@ -36,17 +40,116 @@
     (shift-click or cmd-click to select multiple). Arrow buttons to display below the game canvas.
   = f.collection_select :soft_buttons, soft_button_options, :value, :name, {selected: @level.soft_buttons}, {multiple: true}
 .field
+  = f.label :background, 'Starting background'
+  = f.select :background, options_for_select(@level.class.background_options, @level.background), {include_blank: true}
+.field
   = f.label :edge_collisions, 'Edge collisions'
   = boolean_check_box f, :edge_collisions
 .field
   = f.label :projectile_collisions, 'Projectile collisions'
   = boolean_check_box f, :projectile_collisions
 .field
+  = f.label :item_collisions, 'Item collisions'
+  = boolean_check_box f, :item_collisions
+.field
   = f.label :allow_sprites_outside_playspace, 'Allow sprites outside playspace'
   = boolean_check_box f, :allow_sprites_outside_playspace
 .field
   = f.label :sprites_hidden_to_start, 'Sprites hidden to start'
   = boolean_check_box f, :sprites_hidden_to_start
+.field
```

Comment:
it feels like this could be a bit more data driven.. and then we could experiment with collapsable views or some other way to make the page shorter by default.